apps/src/sites/studio/pages/googleBlocklyWrapper.js

```diff
@@ -0,0 +1,191 @@
+const BlocklyWrapper = function(blocklyInstance) {
```

Comment:
It would be great to have a comment at the top explaining what this module is all about.